src/main/java/com/sprint/mission/discodeit/controller/UserController.java

```diff
@@ -0,0 +1,82 @@
+
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.user.CreateUserRequestDto;
+import com.sprint.mission.discodeit.dto.user.FindUserResponseDto;
+import com.sprint.mission.discodeit.dto.user.UpdateUserRequestDto;
+import com.sprint.mission.discodeit.service.UserService;
+import com.sprint.mission.discodeit.service.UserStatusService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+
+// 사용자 관리 controller
+@RestController
+@RequestMapping("/api/user")
+@RequiredArgsConstructor
+public class UserController {
+
+    private final UserService userService;
+    private final UserStatusService userStatusService;
+
+    // 사용자 등록
+    @RequestMapping(method = RequestMethod.POST)
+    public ResponseEntity<FindUserResponseDto> createUser(@RequestParam("email") String email,
+                                                          @RequestParam("password") String password,
+                                                          @RequestParam("name") String name,
+                                                          @RequestParam("nickname") String nickname,
+                                                          @RequestParam("phoneNumber") String phoneNumber,
+                                                          @RequestParam("profileImageFile") MultipartFile profileImageFile) throws IOException {
```

Comment:
프로필 이미지는 필수가 아니니 아래와 같이 바꿔주는게 좋을 것 같아요.  
```suggestion
                                                          @RequestParam(value = "profileImageFile", required = false) MultipartFile profileImageFile) throws IOException {
```